app/controllers/tariffs_controller.rb

```diff
@@ -42,4 +42,4 @@ def bundle_tariff
   def tariff_params
     params.require(:tariff).permit(:name, :price)
   end
-end
+end
```

Comment:
@romigos https://github.com/rubocop-hq/ruby-style-guide#newline-eof

Reply:
fixed

Reply:
![image](https://user-images.githubusercontent.com/6030146/50875743-67f9e100-13d2-11e9-9d5c-98d76aaab2e2.png)

Reply:
@romigos it is not true

Reply:
fixed